samples/dnn/text_detection.cpp

```diff
@@ -2,17 +2,19 @@
     Text detection model: https://github.com/argman/EAST
     Download link: https://www.dropbox.com/s/r2ingd0l3zt8hxs/frozen_east_text_detection.tar.gz?dl=1
 
-    Text recognition model taken from here: https://github.com/meijieru/crnn.pytorch
-    How to convert from pb to onnx:
-    Using classes from here: https://github.com/meijieru/crnn.pytorch/blob/master/models/crnn.py
```

Comment:
Don't remove existing usage - make sample work with both models.

Reply:
The two text recognition preprocessing is different. Regarding the CRNN in https://github.com/meijieru/crnn.pytorch, I have provided a pre-trained model in my Google Drive. They have the same structure, but different pre-processing. In this case, how should I modify it?

Reply:
So, what's the difference between models? In terms of accuracy/efficiency?

Reply:
Because the model structure is the same, the efficiency is the same.  About the accuracy, since more data for training, my pre-trained model can achieve 89.2% in IC13 dataset, while the original one is 86.7%.

Reply:
so they seems equal that means sample should support both of them. Try to manage it automatically or by flags.

Reply:
Sorry for replying so late, I have added a flag to allow the sample to use both models at the same time.
In addition, the OCR model based on DenseNet+CTC and DenseNet+BiLSTM+CTC has been added https://drive.google.com/drive/folders/1cTbQ3nuZG-EKWak6emD_s8_hHXWz7lAr?usp=sharing.